app/posts/apply-for-teacher-training/user-needs/AN0024.md

```diff
@@ -0,0 +1,7 @@
+---
+title: AN024
+userneed: |
+As a candidate
+I need to get 2 references quickly
+So that I don’t miss out on the course that I want
+---
```

Comment:
```suggestion
userneed: |
  As a candidate
  I need to get 2 references quickly
  So that I don’t miss out on the course that I want
---
```

I think